docs/SNAP_SETUP.md

```diff
@@ -0,0 +1,92 @@
+# Setup
+
+
+To create a new project, we recommend using the [Snapfu CLI](https://www.npmjs.com/package/snapfu) to initialize a new project from a preconfigured scaffold. 
+
+Snapfu is a CLI interface for creating a Snap project from a template. You do not need to utilize Snapfu to develop using Snap, but it does simplify the process and provides additional features. 
+
+
+## Install
+
+Note: We recommend using Node v20 or higher and npm v10 or higher.
+
+```sh
+npm install -g snapfu
+```
+
+
+## Initialize new project
+
+This will create a new project with a git repo from a list of available scaffolds. The `[projectname]` argument is optional and will create a directory with the given name. Otherwise the project will be created in the current directory.
+
+During this you will also be prompted for your Site ID and secret key, which you find in the [Searchspring Management Console](https://manage.searchspring.net)
+
+```sh
+snapfu init [projectname]
+cd [projectname] && npm install
+```
+
+
+## Development
+
+```sh
+npm run dev
+```
+
+This will start the local development server and serve the project build files and the contents of the `public` directory.
+
+| File/Resource                | URL                                                      |
+|------------------------------|----------------------------------------------------------|
+| public/index.html            | [https://localhost:3333/](https://localhost:3333/)       |
+| public/mockup.html (or other files) | [https://localhost:3333/mockup.html](https://localhost:3333/mockup.html) |
+| main bundle                  | [https://localhost:3333/dist/bundle.js](https://localhost:3333/dist/bundle.js) |
+| universal bundle             | [https://localhost:3333/dist/universal.bundle.js](https://localhost:3333/dist/universal.bundle.js) |
```

Comment:
When running `dev` the bundle is here:
https://localhost:3333/universal.bundle.js